src/py/kaleido/_page_generator.py

```diff
@@ -26,6 +26,9 @@ class PageGenerator:
         <script>
           window.PlotlyConfig = {MathJaxConfig: 'local'}
         </script>
+        <script type="text/x-mathjax-config">
+          MathJax.Hub.Config({ "SVG": { blacker: 0 }})
```

Comment:
@ndrezn just wondering if you knew anything about this Config option, its being recomendad for improving some rendering oddities in Kaleido